### DISCUSSION TITLE
After installing I get ERROR 500. Can´t find the issue

### DISCUSSION THREAD

**Discussion by rockandblog** [Laravel-Backpack/community-forum#1146] [Q&A (Help)] [answered]
Hello, I have installed Backpack on Laravel 11. I had done it before without any issues, but this time I’m getting a 500 error when trying to access /admin.

There are no errors appearing in the Laravel log. How can I figure out what’s going on?

As you can see, the installation seems to have gone okay.

Thank you.

```
sudo php artisan backpack:install

   STEP 1  Installing Backpack CRUD.

  Publishing configs, views, routes and assets ........................................................................... DONE
  Creating users table ................................................................................................... DONE
  Creating CheckIfAdmin middleware ....................................................................................... DONE
  Installing Generators .................................................................................................. DONE
  Installing Basset ...................................................................................................... DONE

   STEP 2  Installing a Theme.

  │ Choose your admin UI, depending on your project and preferences.

  Tabler (default) was already installed



   STEP 3  Creating an admin.

  │ Quickly jump in your admin panel, using the email & password you choose here.
  │ Currently there is 1 user in the users table.
  │ Skipping creating an admin user.


   STEP 4  Installing premium Backpack add-ons:

  │ Add tons of features and functionality to your admin panel, using our paid add-ons.
  │ For more information, payment and access please visit https://backpackforlaravel.com/pricing
 DONE
  It was already installed.

  ──────────

  1 Backpack Pro .................................................................................................... INSTALLED
    Adds 5 operations, 10 filters, 28 fields, 6 columns, charts

  2 DevTools .................................................................................................... NOT INSTALLED
    Helps generate models, migrations, operations and CRUDs

  3 Editable Columns ............................................................................................ NOT INSTALLED
    Allow your admins to make small changes from the table view



   INFO  Checking Backpack Basset installation.

  Initializing basset check ................................................................................................................... DONE
  Checking cache storage ...................................................................................................................... DONE
  Fetching a basset ........................................................................................................................... DONE


   DONE  Backpack installation complete.

  │ Go to https://rnbservice.infiniteroles.com/admin/ to access your new admin panel.
  │ You may need to run php artisan serve to serve your Laravel project.
```

**Comment by jcastroa87** [comment]
Hello @rockandblog this fully look like some problem on server, what kind of server are you using? Did you check server errors? 

Let me know if there is something to have a start point.

Cheers.

**Reply by rockandblog** [reply, marked as answer]
It´s very strange, but IO have reinstalled backpack selecting a different UI theme, and no it works :-(

**Reply by jcastroa87** [reply]
I'm sorry, i didn't fully understand, is this working now?

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity, I will close the issue, but please feel free to re-open or create a new one if needed.

Cheers.

**Comment by doncadavona** [comment]
I'm encountering this on `Laravel 12`, `Php 8.4.8` a and Nginx or `php artisan serve`.
Nothing is logged in `storage/logs/*` or Nginx logs.

Debug is set to true and environment is local. Without log or error details, I can't figure out what's wrong.

![image](https://github.com/user-attachments/assets/5d580e9e-acab-48b3-884c-a7d342713933)

**Reply by kleve-r** [reply]
I got the same error when installing.
For debugging, add temporary logging to App\Exceptions\Handler:
`public function render($request, Throwable $exception)
    {
        logger()->error($exception->getMessage() . PHP_EOL . $exception->getTraceAsString());
    }`

My mistake was that I installed the **tabler** theme, and in `config/backpack/ui.php` the default value was used - `coreuiv2` in the `view_namespace` and `view_namespace_fallback` variables

**Reply by nekhbet** [reply]
renaming those settings worked .. but it is kinda strange on a new design to just fail like this

**Reply by karlmarxmanzano** [reply]
@nekhbet thanks confirmed working by changing to tabler if you installed tabler at `/config/backpack/ui.php`.

```    
'view_namespace' => 'backpack.theme-tabler::',
'view_namespace_fallback' => 'backpack.theme-tabler::',
```
